### ISSUE TITLE
EntityNotFoundException when aws_glue_catalog_database is missing

### ISSUE THREAD

**Issue by dotanrs** [terraform-providers/terraform-provider-aws#5136]
Terraform v0.11.5
+ provider.aws v1.26.0

I created a `aws_glue_catalog_database` resource like

```
resource "aws_glue_catalog_database" "aws-glue-database" {
  name  = "${var.db-name}"
}
```
Something happened to the database and it was deleted. Now when I run either terraform apply or destroy, I get

> Error: Error refreshing state: 1 error(s) occurred: <br>
> * module.temp-crawler.aws_glue_catalog_database.aws-glue-database: aws_glue_catalog_database.aws-glue-database: EntityNotFoundException: Database temp not found.
	status code: 400, request id: c7eae1a5-8388-11e8-8e06-ed3b3c0633d6

The expected behavior, IMO, is to simply treat the resource as missing.

This is resolved if I manually create a DB by that name

**Comment by bflad** [comment]
Bug fix pull request submitted: #5141

**Comment by bflad** [comment]
The fix for this has been merged and will release with version 1.27.0 of the AWS provider, likely later today.

**Comment by bflad** [comment]
This has been released in [version 1.27.0 of the AWS provider](https://github.com/terraform-providers/terraform-provider-aws/blob/master/CHANGELOG.md#1270-july-11-2018). Please see the [Terraform documentation on provider versioning](https://www.terraform.io/docs/configuration/providers.html#provider-versions) or reach out if you need any assistance upgrading.

**Comment by Angarsk8** [comment]
Hi @bflad, I'm still having the same issue. I manually deleted the glue database in the AWS console and now I'm trying to create the resource again with a different name, however, everytime I try to apply the changes by running `terraform apply`, it logs the following error in the terminal:

```bash
❯ terraform apply
...
Error: Error refreshing state: 1 error(s) occurred:

* aws_glue_catalog_table.data_glue_schema_table: 1 error(s) occurred:

* aws_glue_catalog_table.data_glue_schema_table: aws_glue_catalog_table.data_glue_schema_table: Error reading Glue Catalog Table: EntityNotFoundException: Database cerberus-somtag-logs not found.
        status code: 400, request id: d2254beb-ecb7-11e8-a254-29691ed6c086
```

I've already upgraded the `aws` plugin provider to version `1.46.0` (which should work as suggested in this thread), however, that didn't work.

**Comment by bflad** [comment]
The similar fix for the `aws_glue_catalog_table` resource has been merged and will release with version 2.6.0 of the Terraform AWS Provider, likely middle of next week.
